tests/utils/getClaimDetails.test.tsx

```diff
@@ -0,0 +1,92 @@
+import getClaimDetails, {
+  buildBenefitYear,
+  formatCurrency,
+  formatDate,
+  getProgramExtensionPair,
+  programExtensionPairs,
+  programExtensionPairType,
+  programTypeNames,
+} from '../../utils/getClaimDetails'
+
+// Test getProgramExtensionPair()
+describe('Converting ProgramType to user-facing strings', () => {
+  it('returns the correct string for each known ProgramType', () => {
+    for (const [id, pair] of Object.entries(programExtensionPairs)) {
+      const parts: programExtensionPairType = getProgramExtensionPair(programTypeNames[id])
+      expect(parts.programType).toBe(pair.programType)
+      expect(parts.extensionType).toBe(parts.extensionType)
+    }
+  })
+
+  it('throws an error if an unknown ProgramType is given', () => {
+    expect(() => {
+      getProgramExtensionPair('unknown')
+    }).toThrowError('Unknown Program Type')
+  })
+})
+
+// Test formatDate()
+describe('Formatting dates', () => {
+  it('displays the expected date string', () => {
+    const notFormatted = '2013-09-27T00:00:00'
+    const formatted = formatDate(notFormatted)
+    expect(formatted).toEqual('9/27/2013')
+  })
+})
+
+// Test buildBenefitYear()
+describe('Constructing the benefit year', () => {
+  it('results in a date range string', () => {
+    const range = buildBenefitYear('2013-09-27T12:34:33', '2021-10-28T05:33:34')
+    expect(range).toBe('9/27/2013 - 10/28/2021')
+  })
+})
+
+// Test formatCurrency()
+describe('Formatting dollar amounts', () => {
+  it('displays the correct string for amounts with cents', () => {
+    const withCents = '532.82'
```

Comment:
Should these be numbers, like the function is expecting, or should the function be taking strings and handling the conversion?

Reply:
I think you can drop all but one test - we only care about checking 'are we getting the output we expect', we don't have to re-test the underlying function!

Reply:
> Should these be numbers, like the function is expecting, or should the function be taking strings and handling the conversion?

Yes! Good catch. I'll fix this.

> I think you can drop all but one test - we only care about checking 'are we getting the output we expect', we don't have to re-test the underlying function!

Fair! I'm always antsy about making sure to test that I've configured the thing to use the underlying function correctly, but I can pare this one down.